bin/gen_csr_access_test.py

```diff
@@ -0,0 +1,225 @@
+import sys
```

Comment:
See the file's [history](https://github.com/openhwgroup/core-v-verif/commits/45e9dd1a6f1c0f3671b3e7b5700ada72fa69e1df/bin/gen_csr_access_test.py) for a list of my changes.